src/platform/i386/kernel.h

```diff
@@ -10,59 +10,25 @@
 #include <thd.h>
 #include <hw.h>
 
-#ifdef ENABLE_CONSOLE
-void vga_clear(void);
-void vga_puts(const char *s);
-void console_init(void);
-#endif
-
-#ifdef ENABLE_VGA
-void vga_init(void);
-void vga_puts(const char *str);
-#endif
-
-#ifdef ENABLE_SERIAL
-void serial_init(void);
-#endif
-
-/* These numbers map directly to actual timers in the HPET */
-typedef enum {
-	TIMER_PERIODIC = 0,
-	TIMER_ONESHOT  = 1,
-} timer_type_t;
-
-#define TIMER_DEFAULT_US_INTERARRIVAL 1000 /* US = microseconds */
-
-void  timer_set(timer_type_t timer_type, u64_t cycles);
-void  timer_init(void);
-u64_t timer_find_hpet(void *timer);
-void  timer_set_hpet_page(u32_t page);
-void  timer_thd_init(struct thread *t);
+#include "vga.h"
+#include "serial.h"
+#include "hpet.h"
+#include "acpi.h"
+#include "lapic.h"
+#include "pic.h"
+#include "ioapic.h"
 
 void  tss_init(void);
 void  idt_init(void);
 void  gdt_init(void);
 void  user_init(void);
+
 void  paging_init(void);
-void *acpi_find_rsdt(void);
-void *acpi_find_timer(void);
-void  acpi_set_rsdt_page(u32_t);
 void  kern_paging_map_init(void *pa);
-
-void *       acpi_find_apic(void);
-u32_t        lapic_find_localaddr(void *l);
-void         lapic_set_page(u32_t page);
-void         lapic_timer_init(void);
-void         lapic_set_timer(int timer_type, cycles_t deadline);
-u32_t        lapic_get_ccr(void);
-void         lapic_timer_calibration(u32_t ratio);
-extern u32_t lapic_timer_calib_init;
-
-void smp_init(void);
+int   vm_set_supage(u32_t addr);
```

Comment:
`supage` -> `superpage`

(`su` also aliases to "super user")

Reply:
kk, I'll rename it.